www/src/data/sidebars/doc-links.yaml

```diff
@@ -218,6 +218,13 @@
           link: /docs/creating-a-sitemap/
         - title: Linking between pages
           link: /docs/linking-between-pages/
+        - title: Adding page transitions
+          link: /docs/adding-page-transitions/
+          items:
+            - title: Adding page transitions with transition-link
```

Comment:
It would help readers to list the entire plugin names here (even if a bit lengthy). i.e.:

"Adding Page Transitions with gatsby-plugin-transition-link"
"Adding Page Transitions with gatsby-plugin-page-transitions*"

Also capitalizing page titles is more consistent with the [Gatsby Style Guide](https://www.gatsbyjs.org/docs/gatsby-style-guide/).